### ISSUE TITLE
[Docs] Outdated installation documentation

### ISSUE THREAD

**Issue by OpNop** [phalcon/docs#858]
While following the Linux install guide [HERE](https://docs.phalconphp.com/en/latest/reference/install.html#linux-solaris) the build failed on my Ubuntu 16.04 VM, after looking around I followed the install instructions from the GIT page and they worked, only difference seems to be the clone command does not have the `--depth=1` on the GIT page as it does on the Docs page.

Also as a side note the Docs page has PHP 5.3 or greater as a prerequisite and for the Solaris packages shows to install PHP5.3, but it seems Phalcon no longer supports 5.3 or 5.4

TLDR:

1. Remove `--depth=1` from clone statement
2. Update Prerequisite
3. Update Solaris packages to newer version of PHP

**Comment by SidRoberts** [comment]
Fixed in 18449a4196dfa0ec93c375cd855d1f43614e5bd1 and b0fe27ec3c90a262e3f5f4f84e56ff62c39def2d.
